Adding bceid lib (#12)
bcgov/spring-boot-starters: src/spring-bceid-starter/README.md

# spring-bceid-starter

> A stater to facilitate usage of bceid service.

Reviewer: ```suggestion
A starter to facilitate usage of the BCeID service.
```

## Usage

Add spring-bceid-starter as a maven dependency

```xml
 <dependencies>
    <dependency>
        <groupId>ca.bc.gov.open</groupId>
        <artifactId>spring-bceid-starter</artifactId>
        <version>0.1.2</version>
    </dependency>
</dependencies>
```

## Configuration

| name | definition | required |
| --- | --- | --- |
| [bcgov.bceid.service.uri](#cgovbceidserviceuri) | String | Yes |
| [bcgov.bceid.service.username](#bcgovbceidserviceusername) | String | Yes |
| [bcgov.bceid.service.password](#bcgovbceidservicepassword) | String | Yes |
| [bcgov.bceid.service.onlineServiceId](#bcgovbceidserviceonlineServiceId) | String | Yes |

#### bcgov.bceid.service.uri

* Value type is String

> Sets the bceid service URI

Reviewer: ```suggestion
Sets the BCeID service URI
```

> Sets the bceid service URI

Author: ```suggestion
Sets the BCeID service URI
```

#### bcgov.bceid.service.username

* Value type is String

> Sets the username used to set basic authentication on bceid service

Reviewer: ```suggestion
Sets the username used to set basic authentication on the BCeID service
```

> Sets the username used to set basic authentication on bceid service

Author: ```suggestion
Sets the username used to set basic authentication on the BCeID service
```

#### bcgov.bceid.service.password

* Value type is String

> Sets the password used to set basic authentication on bceid service

Reviewer: ```suggestion
Sets the password used to set basic authentication on the BCeID service
```

> Sets the password used to set basic authentication on bceid service

Author: ```suggestion
Sets the password used to set basic authentication on the BCeID service
```
